### PR TITLE
Remove all references to `clearText`

### PR DESCRIPTION
**Pull request by amhinson**
This PR updates all docs & types to use `clear` instead of `clearText` for the `Input` component.

### CODE REVIEW

**Comment by amhinson** [comment]
Related issue: https://github.com/react-native-training/react-native-elements/issues/1316

**Comment by iRoachie** [comment]
@amhinson You still need to do a rebase. Your PR has other commits from master. Make sure you base your PR from `next` branch

**Comment by amhinson** [comment]
Ok thanks. updated!

File: website/versioned_docs/version-0.19.0/forms.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ shake the textinput if not a falsy value and different from the previous value
 | shake     | shake the textinput, eg `this.refs.someInputRef.shake()` |
 | focus     | call focus on the textinput ([example](#calling))        |
 | blur      | call blur on the textinput ([example](#calling))         |
-| clearText | call clear on the textinput ([example](#calling))        |
+| clear     | call clear on the textinput ([example](#calling))        |
```
Comment (iRoachie) [inline website/versioned_docs/version-0.19.0/forms.md]: Don't change this. The method in 0.19.0 was actually clearText https://github.com/react-native-training/react-native-elements/blob/v0.19.1/src/form/FormInput.js#L60

File: docs/forms.md [hunk 2]
```diff
@@ -92,7 +92,7 @@ shake the textinput if not a falsy value and different from the previous value
 | shake     | shake the textinput, eg `this.refs.someInputRef.shake()` |
 | focus     | call focus on the textinput ([example](#calling))        |
 | blur      | call blur on the textinput ([example](#calling))         |
-| clearText | call clear on the textinput ([example](#calling))        |
+| clear     | call clear on the textinput ([example](#calling))        |
```
Comment (iRoachie) [inline docs/forms.md]: Don't change this, this file isn't being used anymore.

File: website/versioned_docs/version-0.19.0/forms.md [hunk 3]
```diff
@@ -185,7 +185,7 @@ You can then access FormInput methods like so:
 ```
 this.input.focus();
 this.input.blur();
-this.input.clearText();
+this.input.clear();
```
Comment (iRoachie) [inline website/versioned_docs/version-0.19.0/forms.md]: Same as above, change this back to clearText

File: website/versioned_docs/version-0.19.0/searchbar.md [hunk 4]
```diff
@@ -227,5 +227,5 @@ You can then access SearchBar methods like so:
 ```
 this.search.focus();
 this.search.blur();
-this.search.clearText();
+this.search.clear();
```
Comment (iRoachie) [inline website/versioned_docs/version-0.19.0/searchbar.md]: Change this back to `clearText`, that was the name of the method in 0.19.0 https://github.com/react-native-training/react-native-elements/blob/v0.19.1/src/input/Search.js#L45

File: website/versioned_docs/version-1.0.0-beta5/forms.md [hunk 5]
```diff
@@ -185,7 +185,7 @@ You can then access FormInput methods like so:
 ```
 this.input.focus();
 this.input.blur();
-this.input.clearText();
+this.input.clear();
```
Comment (iRoachie) [inline website/versioned_docs/version-1.0.0-beta5/forms.md]: Don't change this, this file isn't being used anymore.

**Comment by iRoachie** [comment]
Thanks!

**Comment by iRoachie** [comment]
Live now in [1.0.0-beta6](https://github.com/react-native-training/react-native-elements/releases/tag/v1.0.0-beta6)!

**Pull request merged**